app/controllers/notes_controller.rb

```diff
@@ -1,15 +1,19 @@
 class NotesController < ApplicationController
   before_action :find_project
   before_action :find_notes, only: :index
+  after_action :verify_authorized
```

Comment:
Does this work if we put it in ApplicationController 🤔 I suppose not because we have controller with no auth...

Reply:
Sorry to jump in! Whilst this should go into ApplicationController to stop us forgetting, I have just seen how many issues this will raise - we need to swing back over our policies and clear some deby there!

Let's get this work in and write up more detailed work to cover our policies in more depth.

Reply:
Sorry to jump in! Whilst this should go into ApplicationController to stop us forgetting, I have just seen how many issues this will raise - we need to swing back over our policies and clear some deby there!

Let's get this work in and write up more detailed work to cover our policies in more depth.

Reply:
Agree it _should_ go in `ApplicationController`, but also agree that we need to solve policies for all other object types and actions first.